app/Http/Controllers/ClientController.php

```diff
@@ -20,16 +22,17 @@ public function create(): View
         return view('clients.create');
     }
 
-    public function store(Request $request)
+    public function store(StoreClientRequest $storeClientRequest)
     {
-        $requestData = $request->all();
+        $validatedData = $storeClientRequest->validated();
 
         $client = new Client([
-            'name' => $requestData['name'],
-            'surname' => $requestData['surname'],
-            'email' => $requestData['email'],
-            'address' => $requestData['address'],
+        'name' => $validatedData['name'],
```

Comment:
būtu forši tomēr atstāt atstarpes lai būtu "kāpnes" kodā, lai varētu labāk uztvert un izlasīt kodu